packages/caver-account/src/accountKey/accountKeyHelper.js

```diff
@@ -25,6 +28,65 @@ const ACCOUNT_KEY_TAG = {
     ACCOUNT_KEY_ROLE_BASED_TAG: '0x05',
 }
 
+/**
+ * Creates and returns the valid instance of WeightedMultiSigOptions for AccountKeyWeightedMultiSig.
+ * If the user does not define the values of options(threshold, weights),
+ * default options(threshold is 1 and the weight of each key is 1) are returned.
+ *
+ * @param {number} lengthOfKeys The lenght of keys.
+ * @param {WeightedMultiSigOptions|object} [options] An instance of WeightedMultiSigOptions or an object that defines 'threshold' and 'weight'.
+ * @return {WeightedMultiSigOptions}
+ */
+const formatOptionsForMultiSig = (lengthOfKeys, options) => {
+    if (_.isArray(options))
+        throw new Error(`For AccountKeyWeightedMultiSig, options cannot be defined as an array of WeightedMultiSigOptions.`)
+
+    if (!options) options = new WeightedMultiSigOptions(1, Array(lengthOfKeys).fill(1))
+    if (!(options instanceof WeightedMultiSigOptions)) options = WeightedMultiSigOptions.fromObject(options)
+
+    return options.isEmpty() ? new WeightedMultiSigOptions(1, Array(lengthOfKeys).fill(1)) : options
+}
+
+/**
+ * Creates and returns the valid instance of WeightedMultiSigOptions for AccountKeyRoleBased.
+ * If the user does not define the values of options(threshold, weights),
+ * default options(threshold is 1 and the weight of each key is 1) will be used for each role key.
+ *
+ * @param {Array.<number>} lengthOfKeys The lenght of keys.
+ * @param {Array.<WeightedMultiSigOptions>|Array.<object>} [options] An array of WeightedMultiSigOptions or object that defines 'threshold' and 'weight'.
+ * @return {Array.<WeightedMultiSigOptions>}
+ */
+const formatOptionsForRoleBased = (lengthOfKeys, options = []) => {
```

Comment:
How about change method name to "fillWeightedMultiSigOptionForRoleBased"?

Reply:
Sounds good! I changed name to `fillWeightedMultiSigOptionsForRoleBased`